boards/arc/em_starterkit/doc/board.rst

```diff
@@ -63,6 +63,22 @@ which have the EM7D, EM9D, or EM11D SoC.
 Documentation and general information for the board can be found at the
 `embARC website`_, which also includes some free sample software.
 
+
+Supported Firmware Versions
+===========================
+
+The EM Starter Kit has different versions, such as 1.0, 1.1, 2.0, 2.1,
+2.2 and 2.3.
+In Zephyr, 2.2 and 2.3 firmware versions are supported.
+
+* For EM Starter Kit 2.2, EM7D, EM9D and EM11D core configurations are supported.
+  * Use `CONFIG_BOARD_EM_STARTERKIT_R22` to select 2.2 version.
```

Comment:
you can get a hyperlink to the configuration documentation for these options by using

     :option:`CONFIG_BOARD_EM_STARTERKIT_R22`

and do the same for the references to config options below.